src/main/webapp/consentGroup/NewConsentGroupDocuments.js

```diff
@@ -1,14 +1,39 @@
 import { Component, Fragment } from 'react';
 import { WizardStep } from '../components/WizardStep';
-import { hh, h, h1, a, div } from 'react-hyperscript-helpers';
+import { hh, h, h1, a, div, button } from 'react-hyperscript-helpers';
 import { InputFieldFile } from '../components/InputFieldFile';
+import { DOCUMENT_TYPE } from '../util/DocumentType';
+import { AddDocumentDialog } from "../components/AddDocumentDialog";
+import { Panel } from "../components/Panel";
+import { Table } from "../components/Table";
 
+const addDocumentContainer = {
+  display: 'block', height: '40px', marginTop: '15px'
+};
+
+const addDocumentBtn = {
+  position: 'relative', float: 'right'
+};
+
+const headers =
+  [
+    { name: 'Document Type', value: 'fileKey' },
+    { name: 'File Name', value: 'fileName' },
+    { name: '', value: 'remove' }
+  ];
```

Comment:
nit: Since these are all styles, it might make sense to have one const for styles and then have each one of these three const values as an object inside. That would make the code more clearly self-documenting.

Reply:
Oops, this comment applies to `addDocumentContainer` and `addDocumentBtn`, not `headers`, sorry.